File: autoskola-central-front/src/app/core/models/pitanje.ts
import { Odabir } from "./odabir";

export class Pitanje{

    pitanjeId : number | undefined;
    tekstPitanja : String | undefined;
    slika : String | undefined;
    raskrizje : boolean | undefined;
    brojBodova !: number;
    odgovor !: String ;

    odabiri !: Odabir[] ;

}

File: autoskola-central-back/src/main/java/hr/autoskola/model/Pitanje.java
package hr.autoskola.model;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class Pitanje {
	
	private Integer pitanjeId;
	private String tekstPitanja;
	private String slika;
	private Integer raskrizje;
	private Integer brojBodova;
	private String odgovor;
	
	private List<Odabir> odabiri;

}
